src/main/java/com/cryptocurrency/newbchain/Block.java

```diff
@@ -0,0 +1,48 @@
+package com.cryptocurrency.newbchain;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import java.util.Date;
+
+/**
+* This class handles everything related to Blocks that will later constitute a Blockchain.
+*/
+@ToString
+@Getter
+@Setter
+public class Block {
+    
+    private String hash;
+	private String previousHash;
+	private String data; // data will be contain a summary of a transaction.
+	private long timeStamp; 
+    private int nonce;
```

Comment:
More comment on it plz

Reply:
done ![bitmoji](https://sdk.bitmoji.com/render/panel/3d81f2e3-dec8-4395-a284-9c21f770a7e6-a1d78b3b-2b83-4e7e-972d-c729633d7165-v1.png?transparent=1&palette=1&width=246)